### PR TITLE
Clarify doc on multiple .tex files, fix foramtting.

### PR DESCRIPTION
**Pull request by cbf66**

### CODE REVIEW

File: source/help/submit_tex.md [hunk 1]
```diff
@@ -25,14 +25,17 @@ We will soon be rolling out changes to how arXiv process (La)TeX submissions. Th
...
+  
   1. The AutoTeX system would pre-load the LaTeX hyperref package to LaTeX submissions that do not already include it. This package add changes various references to active links in the PDF document (eg, clicking on a reference jumps to the entry in the bibliography section). We will no longer automatically try to add this package. (Note – before you add a \usepackage{hyperref} to your main TeX file, check to see if the template you are using already has a reference by checking the PDF for clickable links; most do).
-1. The AutoTeX system would look for any references in a paper that looked like an arXiv paper ID (such as 	arXiv:2402.08954, or 2402.08954), and turn the into a hyperlink to the paper on arXiv. 
+ 1. The AutoTeX system would look for any references in a paper that looked like an arXiv paper ID (such as 	arXiv:2402.08954, or 2402.08954), and turn the into a hyperlink to the paper on arXiv. 
 We will not do that anymore. Authors should just write out `\href{https://doi.org/10.48550/arXiv.2402.08954}{2402.08954}`. See the [Hyperref documentation for more info](https://mirror.math.princeton.edu/pub/CTAN/macros/latex/contrib/hyperref/doc/hyperref-doc.html).
```
Comment (jweiskoff) [inline source/help/submit_tex.md:34]: why this and not our own canonical url format?

**Review by jweiskoff — CHANGES_REQUESTED** [review]
The input/include section needs work. I'm still angry about this decision as it should have been a part of the original spec for `genpdf`. That aside, the documentation needs updates.

File: source/help/submit_tex.md (hunk 1 above)
Comment (cbf66) [reply source/help/submit_tex.md:35]: The one  we're removing support for?

File: source/help/submit_tex.md [hunk 2]
```diff
@@ -25,14 +25,18 @@ We will soon be rolling out changes to how arXiv process (La)TeX submissions. Th
...
+  
   1. The AutoTeX system would pre-load the LaTeX hyperref package to LaTeX submissions that do not already include it. This package add changes various references to active links in the PDF document (eg, clicking on a reference jumps to the entry in the bibliography section). We will no longer automatically try to add this package. (Note – before you add a \usepackage{hyperref} to your main TeX file, check to see if the template you are using already has a reference by checking the PDF for clickable links; most do).
-1. The AutoTeX system would look for any references in a paper that looked like an arXiv paper ID (such as 	arXiv:2402.08954, or 2402.08954), and turn the into a hyperlink to the paper on arXiv. 
+ 1. The AutoTeX system would look for any references in a paper that looked like an arXiv paper ID (such as 	arXiv:2402.08954, or 2402.08954), and turn the into a hyperlink to the paper on arXiv.
```
Comment (alisonhofer) [inline source/help/submit_tex.md:34]: , and turn the ID into a hyperlink to the paper on arXiv.

File: source/help/submit_tex.md [hunk 3]
```diff
@@ -25,14 +25,18 @@ We will soon be rolling out changes to how arXiv process (La)TeX submissions. Th
...
+      - While changes like these arguably made for a better viewing experience, they also made our TeX processing complex and opaque. When we make modifications to user source like this, the paper on arXiv presents differently than the paper on the user's local machine.
 
-1. arXiv presently has about 70 LaTeX packages we supply that papers can use without needing to upload their own copies. But we have found that only 0.65% of recent submissions depend on this. In some cases these packages are woefully out of date
-   - We are going to eliminate these packages, and provide only those packages that are distributed with each annual TeX Live release. If you use any packages or style files that are not part of Tex Live, please upload them with your submission
+ 1. arXiv presently has about 70 LaTeX packages we supply that papers can use without needing to upload their own copies. But we have found that only 0.65% of recent submissions depend on this. In some cases these packages are woefully out of date
```
Comment (alisonhofer) [inline source/help/submit_tex.md:38]: 1. arXiv presently has about 70 LaTeX packages we supply that papers can use without needing to upload their own copies. We have found that only 0.65% of recent submissions depend on this and, in some cases these packages are woefully out of date.

**Review by alisonhofer — APPROVED** [review]

**Review by jweiskoff — APPROVED** [review]
modulo the one minor quibble with the preamble file, I think this is good (including Alison's changes)

File: source/help/submit_tex.md (hunk 2 above)
Comment (cbf66) [reply source/help/submit_tex.md:34]: Please turn this into a "suggested change" (see item 7 at https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request).

That way I don't to figure exactly what you're suggesting.

File: source/help/submit_tex.md (hunk 3 above)
Comment (cbf66) [reply source/help/submit_tex.md:38]: Please turn this into a "suggested change" (see item 7 at https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request).

That way I don't to figure exactly what you're suggesting.

File: source/help/submit_tex.md [hunk 4]
```diff
@@ -25,14 +25,18 @@ We will soon be rolling out changes to how arXiv process (La)TeX submissions. Th
...
+      - We are going to eliminate these packages, and provide only those packages that are distributed with each annual TeX Live release. If you use any packages or style files that are not part of Tex Live, please upload them with your submission
```
[the remark this answers is not in the document]
Comment (cbf66) [reply source/help/submit_tex.md:39]: Please turn this into a "suggested change" (see item 7 at https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request).

That way I don't to figure exactly what you're suggesting.

File: source/help/00README.md [hunk 5]
```diff
@@ -53,6 +53,16 @@ myMainTexFile.tex toplevelfile
 
 where `myMainTexFile.tex` is the name of the parent TeX file. Note that this does not affect the final assembly order of the final pdf, which is always assembled in alphanumeric order.
 
+Multiple toplevelfile declarations can be used to combine multiple .tex files, each containing a `\documentlcass` command into a single PDF ouput file. For example:
+
+```
+Preamable.tex toplevelfile
```
[the remark this answers is not in the document]
Comment (cbf66) [reply source/help/00README.md:59]: Please turn this into a "suggested change" (see item 7 at https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request).

That way I don't to figure exactly what you're suggesting.

Comment (jweiskoff) [inline source/help/00README.md:59]:
```suggestion
```
Suggestion is to remove this line.

**Review by jweiskoff — CHANGES_REQUESTED** [review]

**Review by jweiskoff — APPROVED** [review]

File: source/help/submit_tex.md (hunk 3 above)
Comment (alisonhofer) [inline source/help/submit_tex.md:38]: arXiv presently has about 70 LaTeX packages we supply that papers can use without needing to upload their own copies. We have found that only 0.65% of recent submissions depend on this and, in some cases these packages are woefully out of date.

File: source/help/submit_tex.md (hunk 4 above)
Comment (alisonhofer) [inline source/help/submit_tex.md:39]: - We are going to eliminate these packages, and provide only those packages that are distributed with each annual TeX Live release. If you use any packages or style files that are not part of Tex Live, please upload them with your submission.

File: source/help/submit_tex.md (hunk 2 above)
Comment (alisonhofer) [inline source/help/submit_tex.md:34]: 1. The AutoTeX system would look for any references in a paper that looked like an arXiv paper ID (such as arXiv:2402.08954, or 2402.08954), and turn the ID into a hyperlink to the paper on arXiv.

**Review by alisonhofer — CHANGES_REQUESTED** [review]

File: source/help/submit_tex.md [hunk 6]
```diff
@@ -25,14 +25,18 @@ We will soon be rolling out changes to how arXiv process (La)TeX submissions. Th
...
+      - If authors need the previous apppend behavior for a submission with multiple `.tex` files, please follow the instructions here: https://info.arxiv.org/help/00README.html.
+      - If image files (JPG, PNG, PDF) files were found, they would be rendered and appended verbatim to the main paper's PDF. If authors want to include images anywhere in their paper, they should use the normal TeX contructs for this. See https://latex-tutorial.com/tutorials/figures/. 
+  
   1. The AutoTeX system would pre-load the LaTeX hyperref package to LaTeX submissions that do not already include it. This package add changes various references to active links in the PDF document (eg, clicking on a reference jumps to the entry in the bibliography section). We will no longer automatically try to add this package. (Note – before you add a \usepackage{hyperref} to your main TeX file, check to see if the template you are using already has a reference by checking the PDF for clickable links; most do).
-1. The AutoTeX system would look for any references in a paper that looked like an arXiv paper ID (such as 	arXiv:2402.08954, or 2402.08954), and turn the into a hyperlink to the paper on arXiv.
```
Comment (alisonhofer) [inline source/help/submit_tex.md:30]: – 1. The AutoTeX system would look for any references in a paper that looked like an arXiv paper ID (such as arXiv:2402.08954, or 2402.08954), and turn the ID into a hyperlink to the paper on arXiv.

File: source/help/submit_tex.md [hunk 7]
```diff
@@ -25,14 +25,18 @@ We will soon be rolling out changes to how arXiv process (La)TeX submissions. Th
...
 We will not do that anymore. Authors should just write out `\href{https://doi.org/10.48550/arXiv.2402.08954}{2402.08954}`. See the [Hyperref documentation for more info](https://mirror.math.princeton.edu/pub/CTAN/macros/latex/contrib/hyperref/doc/hyperref-doc.html).
-   - While changes like these arguably made for a better viewing experience, they also made our TeX processing complex and opaque. When we make modifications to user source like this, the paper on arXiv presents differently than the paper on the user's local machine.
+      - While changes like these arguably made for a better viewing experience, they also made our TeX processing complex and opaque. When we make modifications to user source like this, the paper on arXiv presents differently than the paper on the user's local machine.
 
-1. arXiv presently has about 70 LaTeX packages we supply that papers can use without needing to upload their own copies. But we have found that only 0.65% of recent submissions depend on this. In some cases these packages are woefully out of date
```
Comment (alisonhofer) [inline source/help/submit_tex.md:34]: – 1. arXiv presently has about 70 LaTeX packages we supply that papers can use without needing to upload their own copies. We have found that only 0.65% of recent submissions depend on this and, in some cases these packages are woefully out of date.

File: source/help/submit_tex.md [hunk 8]
```diff
@@ -25,14 +25,18 @@ We will soon be rolling out changes to how arXiv process (La)TeX submissions. Th
...
-   - We are going to eliminate these packages, and provide only those packages that are distributed with each annual TeX Live release. If you use any packages or style files that are not part of Tex Live, please upload them with your submission
```
Comment (alisonhofer) [inline source/help/submit_tex.md:35]: – We are going to eliminate these packages, and provide only those packages that are distributed with each annual TeX Live release. If you use any packages or style files that are not part of Tex Live, please upload them with your submission.

**Review by alisonhofer — CHANGES_REQUESTED** [review]

File: source/help/submit_tex.md (hunk 2 above)
Comment (alisonhofer) [inline source/help/submit_tex.md:34]:
```suggestion
 1. The AutoTeX system would look for any references in a paper that looked like an arXiv paper ID (such as 	arXiv:2402.08954, or 2402.08954), and turn the ID into a hyperlink to the paper on arXiv.
```

File: source/help/00README.md [hunk 9]
```diff
@@ -53,6 +53,15 @@ myMainTexFile.tex toplevelfile
 
 where `myMainTexFile.tex` is the name of the parent TeX file. Note that this does not affect the final assembly order of the final pdf, which is always assembled in alphanumeric order.
```
Comment (alisonhofer) [inline source/help/00README.md:54]:
```suggestion
`myMainTexFile.tex` is the name of the parent TeX file. Note that this does not affect the final assembly order of the final pdf, which is always assembled in alphanumeric order.
```

File: source/help/submit_tex.md [hunk 10]
```diff
@@ -25,14 +25,18 @@ We will soon be rolling out changes to how arXiv process (La)TeX submissions. Th
...
+ 1. arXiv presently has about 70 LaTeX packages we supply that papers can use without needing to upload their own copies. But we have found that only 0.65% of recent submissions depend on this. In some cases these packages are woefully out of date
+      - We are going to eliminate these packages, and provide only those packages that are distributed with each annual TeX Live release. If you use any packages or style files that are not part of Tex Live, please upload them with your submission
```
Comment (alisonhofer) [inline source/help/submit_tex.md:39]:
```suggestion
      - We are going to eliminate these packages, and provide only those packages that are distributed with each annual TeX Live release. If you use any packages or style files that are not part of Tex Live, please upload them with your submission.
```

File: source/help/submit_tex.md [hunk 11]
```diff
@@ -25,14 +25,18 @@ We will soon be rolling out changes to how arXiv process (La)TeX submissions. Th
...
+      - While changes like these arguably made for a better viewing experience, they also made our TeX processing complex and opaque. When we make modifications to user source like this, the paper on arXiv presents differently than the paper on the user's local machine.
 
-1. arXiv presently has about 70 LaTeX packages we supply that papers can use without needing to upload their own copies. But we have found that only 0.65% of recent submissions depend on this. In some cases these packages are woefully out of date
-   - We are going to eliminate these packages, and provide only those packages that are distributed with each annual TeX Live release. If you use any packages or style files that are not part of Tex Live, please upload them with your submission
+ 1. arXiv presently has about 70 LaTeX packages we supply that papers can use without needing to upload their own copies. But we have found that only 0.65% of recent submissions depend on this. In some cases these packages are woefully out of date
```
Comment (alisonhofer) [inline source/help/submit_tex.md:38]:
```suggestion
1. arXiv presently has about 70 LaTeX packages we supply that papers can use without needing to upload their own copies. We have found that only 0.65% of recent submissions depend on this and, in some cases these packages are woefully out of date.
```

**Pull request merged**